wdl_runner/wdl_pipeline.yaml

```diff
@@ -4,12 +4,11 @@ actions:
   entrypoint: ssh-server
   runInBackground: true
   portMappings:
-    22: 22 
+    22: 22
 - containerName: WDL_Runner
   commands: [ '/wdl_runner/wdl_runner.sh' ]
-  imageUri: broadinstitute/wdl-runner:latest
+  imageUri: gcr.io/dg-adam/wdl-runner
```

Comment:
nit: in practice, we wouldn't use this file directly (and need to copy this to our internal repo), so please add a TODO to revert this once your PR has been merged/release upstream.
P.S. until then, we should copy your docker to our internal repository as well.